docs/_Docs_GetStarted.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Get Started
+section: Docs
+order: 2
+---
+
+The Colony Network contracts are moving toward being live on `testnet`, but in the mean time you will need to run your own version for testing and development.
+
+## Set Up and Deploy the colonyNetwork contracts
+
+### Install
+You'll need the latest versions of all the colonyNetwork contracts ready to deploy:
+
+```
+~$ git clone https://github.com/JoinColony/colonyNetwork.git
+
+~$ cd colonyNetwork/
+
+~$ yarn
+
+~$ git submodule update --init
+```
```

Comment:
Can we refer to the readme in `colonyNetwork` itself https://github.com/JoinColony/colonyNetwork#install instead of duplicating it here

Reply:
Would it be alright to refer to the readme, but also keep the info here? The colonyJS and Trufflepig sections below are also in their respective readmes, but this is the only place where they're all together

Reply:
You know we'll forget to update it when the `colonyNetwork` changes but okay